packages/react/src/config.ts

```diff
@@ -1,2 +1,8 @@
 export const CONNECTOR_KEY = 'fuel-current-connector';
 export const BRIDGE_URL = 'https://app.fuel.network/bridge?from=eth&to=fuel';
+export const NATIVE_CONNECTORS = [
```

Comment:
I'd be cleaner to use `.external` from the Fuel SDK version

Reply:
Yes, this is just a implementation before we don't have it.